### DISCUSSION TITLE
Bunit with reqnroll

### DISCUSSION THREAD

**Discussion by KevinTory** [reqnroll/Reqnroll#290] [Q&A] [answered]
I have a Blazor WASM app, and want to use reqnroll with a combination of bunit for simple test cases and Playwright for more complex end-to-end tests. I see broad support for xunit, nunit, etc., but can find nothing for bunit. Is bunit supported?

**Comment by gasparnagy** [comment, marked as answer]
The short answer is no. I have never heard about it so far.

It is possible to make a plugin for Reqnroll to support it if you feel having energy for it.

**Reply by clrudolphi** [reply]
I quickly looked at BUnit. I doubt a plugin approach will work. Both Reqnroll and BUnit generate test classes that then run on one of the main test frameworks.
